values.yaml

```diff
@@ -188,6 +188,14 @@ server:
   # See https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
   updateStrategyType: "OnDelete"
 
+  # Configure the logging verbosity for the Vault server.
+  # Supported log levels include: trace, debug, error, warn, info
+  logLevel: "info"
+
+  # Configure the logging format for the Vault server.
+  # Supported log formats include: standard, json
+  logFormat: "standard"
```

Comment:
Similar thought here, could this default to `""`, and then only set VAULT_LOG_FORMAT env if values.server.logFormat has been set by the user?

Reply:
Fixed as per above.